cgd_tx_eff/src/edu/ohsu/compbio/annovar/annovar_parser.py

```diff
@@ -131,23 +134,11 @@ def _unpack_evf_transcript_tuple(self, delimited_transcript: str):
                 self.logger.debug(f"Converting base pair position to string because it is an offset: {full_c}")
                 hgvs_basep = str(hgvs_basep)
             
-            # p. single letter amino acid; not always present
```

Comment:
Protein information (p. and amino acid position) will no longer read from annovar. They're of no use without a protein transcript, which annovar does not provide.